CarRentals/CarRentalsWebAPI/Controllers/RentalsController.cs

```diff
@@ -0,0 +1,104 @@
+﻿using CarRentals.Models;
+using CarRentalsWebAPI.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CarRentalsWebAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class RentalsController : ControllerBase
+    {
+        private readonly CarRentalsContext _context;
+
+        public RentalsController(CarRentalsContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Rentals
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Rental>>> GetRentals()
+        {
+            return await _context.Rentals.ToListAsync();
+        }
+
+        // GET: api/Rentals/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Rental>> GetRental(int id)
+        {
+            var rental = await _context.Rentals.FindAsync(id);
+
+            if (rental == null)
+            {
+                return NotFound();
+            }
+
+            return rental;
+        }
+
+        // PUT: api/Rentals/5
+        [HttpPut("{id}")]
+        public async Task<IActionResult> PutRental(int id, Rental rental)
+        {
+            if (id != rental.Id)
+            {
+                return BadRequest();
+            }
+
+            _context.Entry(rental).State = EntityState.Modified;
+
+            try
+            {
+                await _context.SaveChangesAsync();
+            }
+            catch (DbUpdateConcurrencyException)
+            {
+                if (!RentalExists(id))
+                {
+                    return NotFound();
+                }
+                else
+                {
+                    throw;
+                }
+            }
```

Comment:
ok, let's see.. an update method usually consist on several steps (most of them were performed by you when using a JSON storage):
- fetch the object to update from the database
- if object does not exist, return something
- if object exists then we proceed to update its properties
- we persist the object in the database

what you have here "works" but it's not ideal nor expected